ast/compile_test.go

```diff
@@ -6728,14 +6756,6 @@ func assertNotFailed(t *testing.T, c *Compiler) {
 	}
 }
 
-func mockStageFunctionCall(c *Compiler) *Error {
```

Comment:
Inlined. These two had only been used once and are simple enough.

Reply:
LGTM. :+1: Nothing complicated going on there, so inlining those functions makes sense.